packages/validator/templates/0.1.0/config.json

```diff
@@ -0,0 +1,253 @@
+{
+	"__arrayItem": {
+		"ois": {
+			"__arrayItem": {
+				"__template": "../1.0.0/ois.json",
+				"apiSpecifications": {
```

Comment:
`apiSpecifications` are already contained in `"./1.0.0/ois.json`

Reply:
Please correct me if I didnt understand whats the problem here, I think you are asking why there is `apiSpecifications` in config template, but also in OIS template. I believe its pretty obvious why is it in OIS, in config it is needed to check if all the security schemes from api specifications are also in `environment.securitySchemes`. This check can't be done in OIS, since it doesnt have full specification so it must be done here.